qemu/tests/qemu_guest_agent.py

```diff
@@ -1580,8 +1580,10 @@ def gagent_check_file_read(self, test, params, env):
         # if guest os resource is enough, will return no error.
         # else it will return error like "insufficient system resource"
         # which is expected
+        # Higher than qemu-5.0(rhel8.3.0), the count number is changed.
+        # Restrict guest-file-read count to 48 MB to avoid crashes.
         try:
-            self.gagent.guest_file_read(ret_handle, count=1000000000)
+            self.gagent.guest_file_read(ret_handle, count=10000000)
```

Comment:
Should we need to consider compatibility with the previous qemu version? 
Suggest that get the current qemu version then compare with `qemu-5.0`, if higher than it` count=10000000` else `count=1000000000`.

Reply:
yes, you should consider qemu-4.0 and the version for rhel7.

Reply:
okay，thanks，I‘ll come up with how to fix it .